build.proj

```diff
@@ -261,6 +262,10 @@
         ExpectedDelaySigned="false"
         ContinueOnError="false" 
         Condition="!$(DelaySign) and '@(DelaySignedAssembliesToSign)' != ''"/>
+		
+	<VerifyAuthenticodeSignatureTask SignedFiles="@(DelaySignedAssembliesToSign);@(ScriptsToSign)">
```

Comment:
All the feedback was incorporated:
1) Now able to verify all Microsoft.*.dll and System.*.dll from the output folder
2) Remove unused code
3) Latest iteration of sign-in job
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/653/consoleFull